migrate_pim/upgrade/upgrade_from_50_to_60.rst

```diff
@@ -0,0 +1,198 @@
+Upgrade from 5.0 to 6.0
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Use this documentation to upgrade projects based on Akeneo PIM Community Edition or Enterprise Edition 5.0 to 6.0.
+
+.. warning::
+    This documentation is in work in progress as the 6.0 version is not available for now
+
+Disclaimer
+**********
+
+Make sure your production database is backed-up before performing the data migration.
+The queue daemon(s) must be stopped as well.
+
+Requirements
+************
+
+Updated System components
+-------------------------
+
+WIP
+
+Updated System dependencies
+---------------------------
+Check your system dependencies are in sync with :doc:`/install_pim/manual/system_requirements/system_requirements`
+
+Updated crontab definition
+--------------------------
+
+Check your crontab is in sync with :doc:`/cloud_edition/flexibility_mode/docs/crontasks`
```

Comment:
```suggestion
Check your crontab is in sync with :doc:`/install_pim/manual/common_install_ce_ee`
```

Reply:
This is a copy-paste from https://github.com/akeneo/pim-docs/blob/master/migrate_pim/upgrade/upgrade_from_40_to_50.rst#updated-crontab-definition  
The doc was not correct?

Reply:
I guess so